recipes/systemd_service.rb

```diff
@@ -0,0 +1,9 @@
+# Our systemd service uses the "simple" service type, so we need
+# stunnel not to fork, and consequently have no need for a pid file.
+node.override['stunnel']['foreground'] = true
+node.override['stunnel']['pid'] = ''
+
+template '/etc/systemd/system/stunnel4.service' do
```

Comment:
Any reason to use this over an actual `systemd_unit` resource? Since you target Chef 13+ it is already available and avoids an extra template. You can even remove the entire recipe call.